pages/_app.tsx

```diff
@@ -2,10 +2,12 @@ import "../styles/globals.css";
 import type { AppProps } from "next/app";
 import { AuthProvider } from "@context/Authentication";
 import { MotionConfig } from "framer-motion";
+import { ToastContainer } from "react-toastify";
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <MotionConfig reducedMotion="user">
+      <ToastContainer containerId="root-toast"/>
```

Comment:
Hello Alex, just checking, this means that if we call toast on this container, it will append the toast to the root of the app correct? Let me know :D

Reply:
Hello, rashad. Yes it will. I hope you can use this later if you need to show toasts. 😁